test/test_jujuversion.py

```diff
@@ -20,28 +20,25 @@
 import ops
 
 
-class TestJujuVersion(unittest.TestCase):
-
-    def test_parsing(self):
-        test_cases = [
-            ("0.0.0", 0, 0, '', 0, 0),
-            ("0.0.2", 0, 0, '', 2, 0),
-            ("0.1.0", 0, 1, '', 0, 0),
-            ("0.2.3", 0, 2, '', 3, 0),
-            ("10.234.3456", 10, 234, '', 3456, 0),
-            ("10.234.3456.1", 10, 234, '', 3456, 1),
-            ("1.21-alpha12", 1, 21, 'alpha', 12, 0),
-            ("1.21-alpha1.34", 1, 21, 'alpha', 1, 34),
-            ("2.7", 2, 7, '', 0, 0)
-        ]
-
-        for vs, major, minor, tag, patch, build in test_cases:
-            v = ops.JujuVersion(vs)
-            assert v.major == major
-            assert v.minor == minor
-            assert v.tag == tag
-            assert v.patch == patch
-            assert v.build == build
+class TestJujuVersion:
```

Comment:
Can we just move all these functions to the top level and avoid the class, especially in cases where the class is basically just the name of the test file?

Reply:
Yes, it makes sense, especially because this test file only has one class to organize all the tests. I will make this change in a single commit.